docs(repos): add repository requirements

TylorS opened the pull request:
Adds a REPOSITORIES.md file which documents what requirements a repository must meet in order to be included into the Motorcycle.js Github organization.
Adds an example .travis.yml file for easy reference.

TylorS committed:
docs(repos): add repository requirements

Adds a REPOSITORIES.md file which documents what a requirements
a repository must meet in order to be included into the Motorcycle.js
Github organization.
Adds an example .travis.yml file for easy reference.

TylorS committed:
docs(repos): fix internal links

Fix links to match those that are generated by Github.

Frikki commented on REPOSITORIES.md:
```diff
@@ -0,0 +1,80 @@
+Required:
+- `eslint` - for running eslint checks
+- `test` - for running tests
+- `start` - must be `npm install && npm prune && validate-commit-msg`
```
If this should work, there’s also a requirement for the dev dependency `validate-commit-msg`.

TylorS replied:
You mean `validate-commit-message` right ?

Frikki replied:
Ah, yes. I have some inconsistency there, and it’s quite confusing.

TylorS committed:
docs(REPOSITORIES): add required and recommended dev depends

Adds required dev Dependency on validate-commit-message for the
use in `npm start` script.
Adds recommended dev dependency on cli-release for the `release-*` npm scripts.
